byoeb-v1/byoeb/byoeb/utils/utils.py

```
@@ -185,4 +192,65 @@ def ensure_utc_dates(obj: Any) -> Any:
+        "response_types": ["code"],
+        "token_endpoint_auth_method": "none",
+        "scope": ",".join(scope.value for scope in scopes)
+    }, timeout=30)
+    reg_resp.raise_for_status()
+    reg = reg_resp.json()
+
+    server_thread = threading.Thread(target=server.handle_request, daemon=True)
+    server_thread.start()
+
+    client = OAuth2Session(reg['client_id'], scope=",".join(scope.value for scope in scopes), redirect_uri=redirect_uri,
+                          code_challenge_method='S256', token_endpoint_auth_method='none')
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "token_endpoint_auth_method"

(S106)